Amazon/Marketplace/v7153/templates/jfrog-artifactory-core-infrastructure.template.yaml

```diff
@@ -0,0 +1,408 @@
+            - !Ref DatabaseEngine
+            - port
+          CidrIp: !Ref PrivateSubnet1Cidr
+        - IpProtocol: tcp
+          FromPort: !FindInMap
+            - DatabaseMap
+            - !Ref DatabaseEngine
+            - port
+          ToPort: !FindInMap
+            - DatabaseMap
+            - !Ref DatabaseEngine
+            - port
+          CidrIp: !Ref PrivateSubnet2Cidr
+        - IpProtocol: tcp
+          FromPort: !FindInMap
+            - DatabaseMap
+            - !Ref DatabaseEngine
+            - port
+          ToPort: !FindInMap
+            - DatabaseMap
+            - !Ref DatabaseEngine
+            - port
+          CidrIp: !Ref PrivateSubnet3Cidr
+      SecurityGroupEgress:
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+          CidrIp: 0.0.0.0/0
+  ArtifactoryS3Bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: Private
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+  ArtifactoryS3IAMPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: S3BucketPermissions
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Sid: S3BucketPermissions
+            Effect: Allow
+            Action:
+              - s3:*
+            Resource:
+              - Fn::Join:
+                - ''
+                - - !Sub "arn:${AWS::Partition}:s3:::"
+                  - !Ref ArtifactoryS3Bucket
+              - Fn::Join:
+                - ''
+                - - !Sub "arn:${AWS::Partition}:s3:::"
+                  - !Ref ArtifactoryS3Bucket
+                  - "/*"
+      Roles:
+        - !Ref ArtifactoryHostRole
+  ArtifactoryEbsVolume:
+    Type: AWS::EC2::Volume
+    Properties:
+      AvailabilityZone:
+        !Select
+        - '0'
+        - !Ref AvailabilityZones
+      Encrypted: false
+      Size: !Ref VolumeSize
+      Tags:
+        - Key: Name
+          Value: !Sub "Artifactory-${AWS::StackName}"
+      VolumeType: gp2
+    DeletionPolicy: Snapshot
+    UpdateReplacePolicy: Snapshot
+
+Outputs:
+  S3Bucket:
+    Value: !Ref ArtifactoryS3Bucket
+    Description: Actual S3 bucket created for Artifactory
+  DatabaseDriver:
+    Value: !FindInMap [DatabaseMap, !Ref DatabaseEngine, Driver]
+  DatabasePlugin:
+    Value: !FindInMap [DatabaseMap, !Ref DatabaseEngine, Plugin]
+  DatabasePluginUrl:
+    Value: !Sub
+      - "${MainURL}${PluginVersion}"
+      - {
+        MainURL: !FindInMap [DatabaseMap, !Ref DatabaseEngine, PluginURL],
+        PluginVersion: !FindInMap [DatabaseMap, !Ref DatabaseEngine, Plugin]
+        }
+  DatabaseType:
+    Value: !FindInMap [DatabaseMap, !Ref DatabaseEngine, Name]
+  DatabaseUrl:
```

Comment:
This is misleading - At first blush I would have thought of a DNS entry or some http. Please call this `JdbcDriverUrl` or something. Also, I am a bit suprised that the formats are identical accros all instances